litequeue.py

```diff
@@ -155,6 +156,9 @@ def __init__(
         memory: bool = False,
         maxsize: Optional[int] = None,
         queue_name: str = "Queue",
+        build_table: bool = True,
+        run_conn_pragmas: bool = True,
+        run_db_pragmas: bool = True,
```

Comment:
I don't think these should be configurable by the user, and should be done by default. And advanced user that really needs a different config can always use the `self.conn` attribute to change the connection as needed.